package.json

```diff
@@ -26,7 +26,8 @@
     "postrelease": "git push --follow-tags origin master",
     "lint": "npm run lint:eslint && npm run lint:stylelint",
     "lint:eslint": "eslint --ignore-path .gitignore .",
-    "lint:stylelint": "stylelint --ignore-path .gitignore \"**/*.css\""
+    "lint:stylelint": "stylelint --ignore-path .gitignore \"**/*.css\"",
+    "docs": "npm run start --prefix docusaurus"
```

Comment:
This is incomplete as we must install deps inside it. Perhaps remove it from this and simply add a section into the root readme about the documentation mentioning we use docsaurus and instructions to run?

Reply:
I believe we can have an instruction to run `npm i --prefix docusaurus` in the `README.md` in the root of the project.

Reply:
Added a note to the root README.